README.md

```diff
@@ -473,7 +473,7 @@ Schedules a job to run `name` once immediately.
 `data` is an optional argument that will be passed to the processing function
 under `job.attrs.data`.
 
-Returns the `job`.
+Saves the job instance and returns a promise which resolves with the `job`.
```

Comment:
@mmiller42 : Can we change the text to `Returns a promise which resolves with the created job or rejects with an error`?